/usr/bin/python3: can't open file '/home/rafa1215/consensus-project/memory/tools/log_geofencing_status.py': [Errno 2] No such file or directory
